### DISCUSSION TITLE
Add current IBM i connection to window title?

### DISCUSSION THREAD

**Discussion by jonii-dev** [codefori/vscode-ibmi#842] [Q&A]
Hi guys!

So I recently started playing around with my window titles (window.titles in settings.json), to faster get to the workspace I want to work with.

I wanted to add the current IBM i connection to the window title and tried it with adding ${remoteName}, but that doesn't seem to work. So this is what it looks like rn:

`${dirty}${activeEditorShort}${separator}${rootName}${seperator}${remoteName}${seperator}VSCode`

Anybody got any idea on how to accomplish this?

Greetings,
Jonii

**Comment by worksofliam** [comment]
Code for IBM i doesn't provide any variables like this to alter the window title. It's something that's been brought up before but never investigated enough.
